app/templates/components/explore/side-bar.hbs

```diff
@@ -0,0 +1,105 @@
+<div class="ui list">
```

Comment:
The dom structure of this is too cluttered. Can we have something very simple but functional and good looking using Semantic UIs vertical menu ? 

Something like this. https://jsfiddle.net/niranjan94/bLbe9ct0/5/ (also note the css added to fix font size)